doc/geany.txt

```diff
@@ -3852,9 +3857,7 @@ Toggle Messages Window                                    Toggles the message wi
 
 Toggle Sidebar                                            Shows or hides the sidebar.
 
-Toggle all additional widgets                             Hide and show all additional widgets like the
-                                                          notebook tabs, the toolbar, the messages window
-                                                          and the status bar.
+Toggle all additional widgets                             See `here <#toggle-all-additional-widgets>`_.
```

Comment:
```suggestion
Toggle all additional widgets                             `Hide and show all additional widgets <#toggle-all-additional-widgets>`_.
```

Reply:
I think the link is enough. Otherwise use 'Show/Hide'.

Reply:
I just don't really like links labeled `here` or `see here`, `click me`, etc., I prefer something more clear.  For the wording, whatever you prefer though.

I don't have a strong opinion though, if you/others prefer it like that let it be so.

Reply:
Agree with @b4n that it should not be "See here" as the sole description of the setting.